http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyHttpServer.java

```diff
@@ -808,8 +809,8 @@ private void registerMicronautChannelHandlers(Map<String, ChannelHandler> channe
             handlers.put(HttpResponseEncoder.ID, responseEncoder);
             handlers.put(NettyServerWebSocketUpgradeHandler.ID, new NettyServerWebSocketUpgradeHandler(
                     nettyEmbeddedServices,
-                    getWebSocketSessionRepository()
-            ));
+                    getWebSocketSessionRepository(),
+                    getApplicationContext().findBean(CoroutineHelper.class)));
```

Comment:
Rather than another bean lookup using `nettyEmbeddedServcies.getRouteExecutor()` and add a getter than returns the CoroutineHelper